src/System.Buffers.Experimental/System/Buffers/BufferExtensions.cs

```diff
@@ -528,5 +528,46 @@ private static Vector<byte> GetVector(byte vectorByte)
                                                        0x01ul << 48) + 1;
         private const ulong byteBroadcastToUlong = ~0UL / Byte.MaxValue;
         private const ulong filterByteHighBitsInUlong = (byteBroadcastToUlong >> 1) | (byteBroadcastToUlong << (sizeof(ulong) * 8 - 1));
+
+
+        /// <summary>
+        /// Determines whether the current span is a slice of the supplied span
+        /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static bool IsSliceOf<T>(this Span<T> child, Span<T> parent) where T : struct
+        {
+            int start; // ignored
+            return child.Length <= parent.Length && IsSliceOf(
+                ref child.DangerousGetPinnableReference(), child.Length,
+                ref parent.DangerousGetPinnableReference(), parent.Length, out start);
+        }
+
+        /// <summary>
+        /// Determines whether the current span is a slice of the supplied span
+        /// </summary>
+        public static bool IsSliceOf<T>(this Span<T> child, Span<T> parent, out int start) where T : struct
+        {
+            start = -1;
```

Comment:
Hmm... in that case maybe we set it to default(int)? Assigning as -1 makes it seem like -1 has some special semantics in the usage.